packages/app-store/zapier/components/AppSettingsInterface.tsx

```diff
@@ -58,6 +58,12 @@ const templates: Template[] = [
     text: "Add new bookings to Google Calendar",
     link: "https://zapier.com/app/editor/template/1083651",
   },
+  {
+    icon: "googleCalendar.svg",
+    app: "Google Calendar",
+    text: "Add new calendar event when a team member is out of office",
+    link: "https://zapier.com/app/editor/template/ToBeDefined",
```

Comment:
Once you got created this template in your app update ID here

Reply:
lets follow up with this in a different PR so we can merge this one